pkg/rbd/rbd_util.go

```diff
@@ -550,6 +550,25 @@ func createSnapshot(pOpts *rbdSnapshot, adminID string, credentials map[string]s
 	return nil
 }
 
+// flattenImage remove the reference from the child clone to the parent snapshot
```

Comment:
@Madhu-1 its also good to refer the documentation here in the source code comment.